lib/iris/experimental/geobridge.py

```diff
@@ -0,0 +1,70 @@
+from geovista import Transform
+from geovista.common import VTK_CELL_IDS
+
+from iris.exceptions import CoordinateNotFoundError
+from iris.experimental.ugrid import Mesh
+
+
+def _get_coord(cube, axis):
+    try:
+        coord = cube.coord(axis=axis, dim_coords=True)
+    except CoordinateNotFoundError:
+        coord = cube.coord(axis=axis)
+    return coord
+
+
+def cube_faces_to_polydata(cube):
+    if cube.mesh:
+        lons, lats = cube.mesh.node_coords
+        face_node = cube.mesh.face_node_connectivity
+        indices = face_node.indices_by_location()
+
+        polydata = Transform.from_unstructured(
+            lons.points,
+            lats.points,
+            indices,
+            data=cube.data,
+            name=f"{cube.name()} / {cube.units}",
```

Comment:
```suggestion
            name=f"{cube.name()} / ({cube.units})",
```

I just checked what it looks like when printing a `Cube`.